blockchain-scrapers/cmd/dash/dash.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/diadata-org/api-golang/dia"
+)
+
+const (
+	HOST     = "localhost"
```

Comment:
Hello, I think you forgot to add a dash node in docker-compose and then this should point to it and not localhost.

Reply:
@frranck Okay, I updated the host.